README.md

```diff
@@ -227,34 +227,58 @@ var playerConfig = {
 } 
 
 ```
-## Telemetry property description
-|Property Name| Description| Default Value
+
+## Metadata Mandatory property description
+|Property Name| Description|  Mandatory/Optional|
 |--|----------------------|--|
-| `context` | It is an `object` it contains the `uid`,`did`,`sid`,`mode` etc., these will be logged inside the telemetry  | ```{}``` |
-| `mode` | It is  `string` to identify preview used by the user to play/edit/preview | ```play```|
-| `authToken` | It is  `string` and Auth key to make  api calls | ```''```|
-| `sid` | It is  `string` and User sessionid on portal or mobile | ```''```|
-| `did` | It is  `string` and Unique id to identify the device or browser| ```''```|
-| `uid` | It is  `string` and Current logged in user id| ```''```|
-| `channel` | It is `string` which defines channel identifier to know which channel is currently using.| `in.sunbird` |
-| `pdata` | It is an `object` which defines the producer information it should have identifier and version and canvas will log in the telemetry| ```{'id':'in.sunbird', 'ver':'1.0'}```|
-| `contextRollup` | It is an `object` which defines content roll up data | ```{}```|
-| `tags` | It is an `array` which defines the tag data | ```[]```|
-| `objectRollup` | It is an `object` which defines object rollup data | ```{}```|
-| `host` | It is  `string` which defines the from which domain content should be load|```window.location.origin```  |
-| `userData` | It is an `object` which defines user data | ```{}```|
-| `cdata` | It is an `array` which defines the correlation data | ```[]```|
+| `identifier` | It is  `string` of uniq content id | Mandatory |
+| `Name` | It is  `string` to represent the name of the content or pdf | Mandatory |
+| `artifactUrl` | It is  `string` url  to load the pdf from artifact url | Mandatory |
+| `streamingUrl` | It is  `string` url  to load the pdf from streaming url | Optional |
+| `isAvailableLocally` | It is a `boolen` value which indicate the content is locally available | Optional |
+| `basePath` | It is `string` to represent the base path of the pdf file | Optional |
+| `baseDir` | It is `string` to represent the base path of the pdf file | Optional |
+| `compatibilityLevel` | It is `number` to represent the compatibility level | Optional |
+| `pkgVersion` | It is `number` to represent the version of the current packages | Optional |
```

Comment:
What is the default value if this is optional or behaviour should be updated here and all other optional fields

Reply:
Updated READMD file